cli/cli_test.go

```diff
@@ -35,16 +35,32 @@ import (
 
 	"github.com/ipfs/go-cid"
 	"github.com/ipfs/go-datastore"
+	"github.com/ipfs/go-datastore/fs"
 	"github.com/meta-network/go-meta"
 )
 
 // TestCWRCommands tests running the 'meta cwr convert' and
 // 'meta cwr index' commands.
 func TestCWRCommands(t *testing.T) {
-	c := newTestCLI(t)
-
+	// create a path to store the index and to store the meta objects.
+	tmpDir, err := ioutil.TempDir("", "meta-main-test")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.RemoveAll(tmpDir)
+	//use fs datastore because the in mempory map data store is not a thread-safe
+	store, err := fs.NewDatastore(tmpDir)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Should we not just do all of this in `newTestCLI`?

Reply:
the newTestCLI is used by the ERN test which is using in memory datastore.
Should we changed that ? or pass a param for the datastore type ?

Reply:
Let's use the fs store for all the tests, we need to make al the converters concurrent eventually.